src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -407,57 +550,57 @@ private void updateTooltipTexts() {
     }
 
     private void populateJobBox() {
-        jobBox.getItems().clear();
-
-        for (GameJob job : GameJob.values()) {
-            if (job.isOnlyInstalled() && (launcherSettings.getJob() != job)) {
-                boolean foundInstalled = false;
-                final List<TerasologyGameVersion> gameVersionList = gameVersions.getGameVersionList(job);
-                for (TerasologyGameVersion gameVersion : gameVersionList) {
-                    if (gameVersion.isInstalled()) {
-                        foundInstalled = true;
-                        break;
-                    }
-                }
-                if (!foundInstalled) {
-                    continue;
-                }
-            }
-
-            final JobItem jobItem = new JobItem(job);
-            jobBox.getItems().add(jobItem);
-            if (launcherSettings.getJob() == job) {
-                jobBox.getSelectionModel().select(jobItem);
-            }
-        }
-
-        updateBuildVersionBox();
-
-        // add change listeners
-        jobBox.getSelectionModel().selectedItemProperty().addListener(new ChangeListener<JobItem>() {
-            @Override
-            public void changed(final ObservableValue<? extends JobItem> observableValue, final JobItem oldItem, final JobItem newItem) {
-                if (jobBox.getItems().isEmpty()) {
-                    return;
-                }
-                launcherSettings.setJob(newItem.getJob());
-                updateBuildVersionBox();
-                updateGui();
-                logger.debug("Selected gamejob: {} -- {}", launcherSettings.getJob(), launcherSettings.getBuildVersion(launcherSettings.getJob()));
-            }
-        });
-
-        buildVersionBox.getSelectionModel().selectedItemProperty().addListener(new ChangeListener<VersionItem>() {
-            @Override
-            public void changed(final ObservableValue<? extends VersionItem> observableValue, final VersionItem oldVersionItem, final VersionItem newVersionItem) {
-                if (newVersionItem != null) {
-                    final Integer version = newVersionItem.getVersion();
-                    launcherSettings.setBuildVersion(version, launcherSettings.getJob());
-                    logger.debug("Selected gamejob: {} -- {}", launcherSettings.getJob(), launcherSettings.getBuildVersion(launcherSettings.getJob()));
-                    updateGui();
-                }
-            }
-        });
+//        jobBox.getItems().clear();
```

Comment:
Commented code should be removed. If the method is no longer used, all usages should be removed/replaced